### DISCUSSION TITLE
how to verify if expected error message is returned in a test file?

### DISCUSSION THREAD

**Discussion by zippond** [duckdb/duckdb#3716] [Q&A] [answered]
If I want to make sure the error msg is "Error: Parser Error: SELECT INTO not supported!" for below query:
`SELECT * INTO t2 FROM t WHERE t LIKE 'b%';`
How to verify it in a test file?

**Comment by Mytherin** [comment, marked as answer]
That is not supported in the test suite right now.

**Reply by zippond** [reply]
Thanks.
